ember-bootstrap/addon/components/bs-link-to.hbs

```diff
@@ -1,10 +1,8 @@
-{{! @glint-nocheck }}
 <LinkTo
   @route={{@route}}
   @models={{this.models}}
   @query={{this.query}}
   @disabled={{@disabled}}
-  class={{@class}}
```

Comment:
`<BsLinkTo>` is a private component. We didn't used that argument anywhere. So I think we can just drop it. Likely a left-over from a very old version.

Reply:
Please ignore. Tests are telling me that I haven't searched correctly. Will fix.